### PR TITLE
Update parse SDK to 1.9.1.

### PR DESCRIPTION
**Pull request by jacob-meacham**
The specific issue this fixes for us is https://github.com/ParsePlatform/Parse-SDK-JS/issues/311.

I've only done smoke testing of this fix, but will be deploying it to our staging instance today and will update if I run into any issues.

### CODE REVIEW

**Comment by TylerBrock** [comment]
Any updates on this one @jacob-meacham ?

**Comment by jacob-meacham** [comment]
We've been using it in staging and haven't seen any issues - however, this issue was wrapped up in another issue I was having trying to wrangle binary uploads/downloads through AWS API Gateway (we've since moved away from proxying through APIG). 

I believe that this PR is still required, but given the other issue I was having, it might not be.

**Comment by aontas** [comment]
This will also fix https://github.com/ParsePlatform/parse-dashboard/issues/618.

File: src/dashboard/Dashboard.js [hunk 1]
```diff
@@ -163,8 +163,8 @@ class Dashboard extends React.Component {
         }
       });
       return Parse.Promise.when(appInfoPromises);
-    }).then(function() {
-      Array.prototype.slice.call(arguments).forEach(app => {
+    }).then(function(resolvedApps) {
+      Array.prototype.slice.call(resolvedApps).forEach(app => {
```
Comment (flovilmart) [inline src/dashboard/Dashboard.js]: Is resolvedApps is an array, then no need to call Array.prototype.slice on it.

**Comment by jacob-meacham** [comment]
@flovilmart Sorry, I didn't see that this was waiting on changes from me!

**Comment by flovilmart** [comment]
LGTM!

**Pull request merged**